tests/unit/RecordUnitTests.d

```diff
@@ -524,3 +524,80 @@ D");
 
 }
 
+@("issue #166 - translate record with name from macro")
```

Comment:
There's a specific directory with tests for GitHub issues [1]. @ciechowoj I'm not sure of the structure of the tests, when to create a separate file for an issue and when not to.

[1] https://github.com/jacob-carlborg/dstep/tree/master/tests/unit/issues

Reply:
@jacob-carlborg It's you who suggested to put the issues for the tests there, if I remember correctly : ), so maybe for consistency @atilaneves you can put your test there as well...

Reply:
Hehe, I don't remember anymore 😃.